### DISCUSSION TITLE
v1.5.7-beta

### DISCUSSION THREAD

**Discussion by albertpb** [albertpb/sd-manager#28] [General]
- Add scan models buttons in settings
- Revert change for decode paths in models
- Fix drag and drop icon not found when packaged
- Add bar with images and models counter
- Fix not closing delete images dialog

<hr /><em>This discussion was created from the release <a href='https://github.com/albertpb/sd-manager/releases/tag/v1.5.7'>v1.5.7-beta</a>.</em>

**Comment by AsuraAce** [comment]
just my personal opinion but i think it would be cleaner and better looking if you replaced the sidebar with something like this in the window of each tab itself. just as an example:

![image](https://github.com/albertpb/sd-manager/assets/13394741/f6e80443-bcbe-4d94-a762-4704be85b94c)

- sort options could be consolidated into a drop down into each respective window / tab itself 
- seems to be true for toggle options as well (doesn't need to be a drop down)
- delete images is destructive and probably shouldn't be exposed in a sidebar
- update database could either be removed (it still exist in the settings) or put into the window as well.
